src/Statistics.jl

```diff
@@ -1142,4 +1142,17 @@ function centralize_sumabs2!(R::AbstractArray{S}, A::SparseMatrixCSC{Tv,Ti}, mea
     return R
 end
 
+##### Sampling #####
+
+"""
+    sample([rng], a)
+
+Uniformly select a single random element of `a`.
+
+Optionally specify a random number generator `rng` as the first argument
+(defaults to `Random.GLOBAL_RNG`).
+"""
+sample(rng::AbstractRNG, a::AbstractArray) = a[rand(rng, eachindex(a))]
```

Comment:
Actually why not call `rand` directly on `a`?
```suggestion
sample(rng::AbstractRNG, a::AbstractArray) = rand(rng, a)
```

This stresses even more that without more methods `sample` isn't very useful (apart from defining the function in the stdlib). We should probably move `sample([rng], a, n::Integer; replace=true, ordered=false)` to Statistics soon after this PR.